ngi_pipeline/engines/sarek/models/sarek.py

```diff
@@ -529,9 +529,10 @@ def generate_tsv_file_contents(self, analysis_sample):
         """
         rows = []
         patientid = analysis_sample.sampleid
-        gender = "ZZ"
+        gender = "NA"
         status = 0
         sampleid = patientid
+        rows.append(['patient','sex','status','sample','lane','fastq_1','fastq_2'])
```

Comment:
do you need to remove something when you add this?

Reply:
The earlier input tsv file didn't have headers. The latest sarek expects them

Reply:
"the earlier input tsv file didn't have headers. The latest sarek expects them"

Reply:
lol